src/main/java/org/broadinstitute/hellbender/tools/walkers/variantutils/ReblockGVCF.java

```diff
@@ -568,8 +566,7 @@ VariantContext cleanUpHighQualityVariant(final VariantContext variant) {
         if(allelesNeedSubsetting && !keepAllAlts) {
             newAlleleSetUntrimmed.removeAll(allelesToDrop);
             final GenotypesContext gc = AlleleSubsettingUtils.subsetAlleles(variant.getGenotypes(), genotype.getPloidy(), variant.getAlleles(),
-                    newAlleleSetUntrimmed, null, GenotypeAssignmentMethod.USE_PLS_TO_ASSIGN,
-                    variant.getAttributeAsInt(VCFConstants.DEPTH_KEY, 0), true);
+                    newAlleleSetUntrimmed, null, GenotypeAssignmentMethod.USE_PLS_TO_ASSIGN);
```

Comment:
Add a comment documenting why we're using `GenotypeAssignmentMethod.USE_PLS_TO_ASSIGN` here instead of the `GenotypeAssignmentMethod.BEST_MATCH_TO_ORIGINAL` we're using everywhere else.